arangosh/Shell/V8ClientConnection.cpp

```diff
@@ -64,6 +64,19 @@ using namespace arangodb::import;
 
 namespace fu = arangodb::fuerte;
 
+namespace {
+// return an identifier to a connection configuration, consisting of
+// endpoint, username, password, jwt, authentication and protocol type
+std::string connectionIdentifier(fuerte::ConnectionBuilder& builder) {
+  return
+      builder.normalizedEndpoint() + "/" +
+      builder.user() + "/" + builder.password() + "/" + 
+      builder.jwtToken() + "/" +
+      to_string(builder.authenticationType()) + "/" + 
+      to_string(builder.protocolType());
```

Comment:
I would prefer a struct with separate fields over a concatenated string, but that is primarily a matter of taste,